roadmap.md

```diff
@@ -13,7 +13,8 @@ The Microsoft.Data.SqlClient roadmap communicates project priorities for evolvin
 |---------------------------|--------------|---------------|
 | Microsoft.Data.SqlClient v1.1 (servicing) | As needed (see also [1.1 releases](https://github.com/dotnet/sqlclient/blob/master/release-notes/1.1)) | Closed |
 | Microsoft.Data.SqlClient v2.1 (servicing) | As needed (see also [2.1 releases](https://github.com/dotnet/sqlclient/blob/master/release-notes/2.1)) | Closed |
-| Microsoft.Data.SqlClient v3.0 | GA (General Availability) estimated for May 2021 | [SqlClient 3.0.0](https://github.com/dotnet/SqlClient/projects/7) |
+| Microsoft.Data.SqlClient v3.0 | GA (General Availability) June 2021 | [SqlClient 3.0.0](https://github.com/dotnet/SqlClient/projects/7) | Closed |
+| Microsoft.Data.SqlClient v4.0 | GA (General Availability) estimated for November 2021 | [SqlClient 4.0.0](https://github.com/dotnet/SqlClient/projects/8)
```

Comment:
That's a very short window between major releases. 
I have to assume that there's some private justification for the version bump that isn't clear from the codebase at this time.

Reply:
Is is not due to many breaking changes?

Reply:
6 months is our regular pace of a GA release but due to important breaking changes necessary this time, it required a major version bump.